.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,72 @@
+---
+name: Bug report
+about: Report a bug within the deCONZ REST-API
+title: ''
+labels: Bug report
+assignees: ''
+
+---
+
+<!--
+  - Use this issue template to report a bug in the deCONZ REST-API.
+  - If you want to report a bug for the Phoscon App, please head over to: https://github.com/dresden-elektronik/phoscon-app-beta
+  - If you're unsure if the bug fits into this issue tracker, please ask for advise in our Discord chat: https://discord.gg/QFhTxqN
```

Comment:
Create a wiki page to document what other systems and their respective github to report issues on if it's not a deconz issue. Users should be able to find out themselves without the need to ask someone

Reply:
Indeed this would be really helpful